src/libdredd/src/mutation_replace_binary_operator.cc

```diff
@@ -59,6 +61,47 @@ std::string MutationReplaceBinaryOperator::GetExpr(
   return result;
 }
 
+bool MutationReplaceBinaryOperator::IsRedundantReplacementOperator(
+    clang::BinaryOperatorKind op, clang::ASTContext& ast_context) const {
+  clang::Expr::EvalResult eval_result;
+  bool rhs_is_int =
+      binary_operator_.getRHS()->EvaluateAsInt(eval_result, ast_context);
+
+  // In the following cases, the replacement is equivalent to either replacement
+  // with a constant or argument replacement.
+  // See https://github.com/mc-imperial/dredd/issues/89 for a more in-depth
+  // explanation.
```

Comment:
I would get rid of this reference to the issue. But I think it would be worth adding brief comments to note what each case is covered by.

Reply:
Done.